stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -296,7 +296,7 @@ internal func reflect(instanceAddress: UInt, kind: InstanceKind) {
 /// This reflects the stored properties of the immediate class.
 /// The superclass is not (yet?) visited.
 public func reflect(object: AnyObject) {
-  let address = unsafeAddress(of: object)
+  var address = Unmanaged.passUnretained(object).toOpaque()
```

Comment:
This two pieces of code in `SwiftReflectionTest.swift` look buggy to me.  They need at least a `fixLifetime`, could you add it?  (`defer { _fixLifetime(object) }` at the beginning of the method.)